source/deploying-with-ansible/guide/install-wazuh-manager.rst

```diff
@@ -218,14 +239,20 @@ We can check the status of our new services in our Wazuh server.
 .. code-block:: console
 
 	[root@localhost centos]# systemctl status wazuh-api
+
+.. code-block:: none
+	:class: output
+
 	● wazuh-api.service - Wazuh API daemon
 	   Loaded: loaded (/etc/systemd/system/wazuh-api.service; enabled; vendor preset: disabled)
 	   Active: active (running) since jue 2018-09-13 12:36:54 CEST; 36min ago
 
 - Filebeat.
 
-.. code-block:: console
+.. code-block:: none
+	:class: output
 
+	[root@localhost centos]# systemctl status filebeat
```

Comment:
I think this line is incorrectly placed inside a code-block for output.

Reply:
It's true it should be outside the output. Fixed!